docs/mp/persistence.adoc

```diff
@@ -0,0 +1,1430 @@
+                <enableLazyInitialization>true</enableLazyInitialization>
+            </configuration>
+        </execution>
+    </executions>
+</plugin>
+----
+
+You must also
+https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#appendix-legacy-bootstrap:~:text=Place%20a%20file%20named%20hibernate.properties%20in%20a%20root%20directory%20of%20the%20classpath[add
+a file named `src/main/resources/META-INF/hibernate.properties`] with
+the following line as its sole contents:
+
+.`src/main/resources/META-INF/hibernate.properties`
+[source,properties]
+----
+hibernate.bytecode.provider=none # <1>
+----
+<1> This instructs Hibernate ORM to
+    https://github.com/hibernate/hibernate-orm/blob/6.1.1/hibernate-core/src/main/java/org/hibernate/cfg/Environment.java#L236[not
+    attempt to perform weaving at runtime].
+
+For more on the `hibernate-enhance-maven-plugin` in particular, see
+https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#tooling-maven[its
+documentation].
+
+For more on Hibernate ORM's bytecode enhancement (weaving) in general,
+see
+https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#BytecodeEnhancement[Bytecode
+Enhancement] in Hibernate ORM's documentation.
+
+For more on bytecode enhancement properties, see
+https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#configurations-bytecode-enhancement[Bytecode
+Enhancement Properties] in Hibernate ORM's documentation.
+
+===== Maven Coordinates (Eclipselink) [[JPA-Eclipselink-Maven-Coordinates]]
+
+To include Helidon's Jakarta Persistence-related integration for
+Eclipselink:
+
+* xref:../about/managing-dependencies.adoc[Ensure your dependencies are
+managed]
+
+* <<JPA-Maven-Coordinates,Ensure the basics of your JPA project are
+  set up properly>>
+
+* Ensure the following `<dependency>` elements are present as child
+  elements of your project's `pom.xml` file's `<dependencies>`
+  element:
++
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.integrations.cdi</groupId>
+    <artifactId>helidon-integrations-cdi-eclipselink</artifactId>
+    <scope>runtime</scope> <!--1-->
+</dependency>
+----
+<1> The `scope` is `runtime`, which ensures that Helidon MP's
+    Eclipselink integration is available at runtime.
+
+===== Setting Up Static Weaving (Eclipselink) [[JPA-Eclipselink-Weaving]]
+
+Eclipselink can alter your classes' bytecode at build time to keep
+track of changes made to objects participating in Jakarta Persistence
+workflows.
+
+To set up this required static weaving for Eclipselink, ensure that
+the following `<plugin>` element is present as a child element of your
+project's `pom.xml` file's `<plugins>` element:
+
+[source,xml]
+----
+<plugin>
+    <groupId>org.codehaus.mojo</groupId>
+    <artifactId>exec-maven-plugin</artifactId>
+    <!--
+        Ideally, your plugin versions are managed via a
```

Comment:
Omitting the plugin version is fine, hard-coding to an arbitrary version is also fine, we don't have to explain how to manage Maven plugins in the persistence documentation.